### ISSUE TITLE
relax_column_count - more granular control

### ISSUE THREAD

**Issue by falkenhawk** [adaltas/node-csv-parse#268]
**Is your feature request related to a problem? Please describe.**
In my case I'd like to allow records with less columns than expected, but throw an error on rows with more columns than expected.

**Describe the solution you'd like**
additional options, next to `relax_column_count` e.g.:
 - `allow_more_column_count` - allow records with more columns than expected,  
 - `allow_less_column_count` - allow records with less columns than expected,

**Describe alternatives you've considered**
At the moment I used version from `master` and hacked it inside `on_record` option, using internal `parser.state` and `parser.__error`...

**Additional context**

Example:
csv headers: username,email,first_name,last_name,roles,groups
Suppose first two columns are required, the rest is optional, so I'd like to allow csvs like
```
user1;user1@example.com
user2;user2@example.com
```
but throw an error for a row like
```
user2323;user2323@example.com;john;doe;admin;local;some_other_value;etc
```
- as the importer has no clue what to do there - might be an user mistake

**Comment by wdavidw** [comment]
Thank you for your suggestion, new options `relax_column_count_less` `relax_column_count_more` are introduce in version 4.8.0

**Comment by falkenhawk** [comment]
❤️

**Comment by falkenhawk** [comment]
@wdavidw the new options are missing from `parse.Options` in `index.d.ts` unfortunately

**Comment by wdavidw** [comment]
Fixed in 4.8.2, thank you.
